CSS/Styles.css

```diff
@@ -0,0 +1,96 @@
+* {
+  width: 320px;
```

Comment:
Please remove this `width`, it is causing your layout to have a strange behavior that doesn't match the template design. This is what I mean:
<img width="245" alt="Screen Shot 2022-05-25 at 17 16 28" src="https://user-images.githubusercontent.com/82426682/170376836-58cfe8e8-4733-46a9-85dd-8d612e3fad1a.png">

I recommend you don't set a fixed `width` for your `body` unless it is of `100%` or `100vw`.

Reply:
Thanks for the suggestion.